<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Perforce::Internal::SettingsPage</class>
 <widget class="QWidget" name="Perforce::Internal::SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>366</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>P4 command:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Utils::PathChooser" name="pathChooser" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="environmentGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Environment Variables</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>P4 port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="portLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="clientLabel">
        <property name="text">
         <string>P4 client:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="clientLineEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="userLabel">
        <property name="text">
         <string>P4 user:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="userLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="logCountLabel">
        <property name="text">
         <string>Log count:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="logCountSpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="timeOutLabel">
        <property name="text">
         <string>Timeout:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="timeOutSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>396</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="promptToSubmitCheckBox">
        <property name="text">
         <string>Prompt on submit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QCheckBox" name="autoOpenCheckBox">
        <property name="text">
         <string>Automatically open files when editing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="text">
        <string notr="true" extracomment="Placeholder">errorLabel: blah blubb</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="testPushButton">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>environmentGroupBox</tabstop>
  <tabstop>portLineEdit</tabstop>
  <tabstop>clientLineEdit</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>logCountSpinBox</tabstop>
  <tabstop>timeOutSpinBox</tabstop>
  <tabstop>promptToSubmitCheckBox</tabstop>
  <tabstop>autoOpenCheckBox</tabstop>
  <tabstop>testPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
